microsoft-edge/webview2/get-started/winui.md

```diff
@@ -9,34 +9,14 @@ ms.prod: microsoft-edge
-
-## Prerequisites  
-
-Ensure you install the following list of pre-requisites before proceeding.  
-
-*   [WebView2 Runtime][Webview2Installer] or any [Microsoft Edge (Chromium) non-stable channel][MicrosoftedgeinsiderDownload] installed on Windows 10 version 1803 \(build 17134\) or later.  For more information about Windows 10, navigate to [Windows Update: FAQ][MicrosoftSupport12373].  
-    
-    > [!NOTE]
-    > The WebView team recommends using the Canary channel and the minimum required version is 82.0.488.0.  
-    
-*   [Visual Studio][MicrosoftVisualstudioMain] 2019, version 16.9 Preview.  For more information, navigate to [Windows UI Library 3 Preview 3][WindowsAppsWinui3ConfigureYourDevEnvironment].  
-    *   Include the following workloads when you install Visual Studio.  
-        *   .NET Desktop Development \(the installer also installs .NET 5\)  
-        *   Universal Windows Platform development  
-    *   To build C++ apps, you must also include the following workloads.  
-        *   Desktop development with C++  
-        *   The C++ \(v142\) Universal Windows Platform tools optional component for the Universal Windows Platform workload.  For more information,  navigate to **Installation Details** under the **Universal Windows Platform development** section, on the right pane.  
         
-## Step 0 - Visual Studio settings  
+## Step 0 - Set Up Development Environment 
 
-1.  Ensure your system has a NuGet package source enabled for [nuget.org][NugetHome].  For more information, navigate to [Common NuGet configurations][NugetConsumePackagesConfiguringNugetBehavior] and [Windows Community Toolkit][WindowsCommunitytoolkit].  
-1.  Download and install the [Project Reunion VSIX package][VisualstudioMarketplaceProjectreunionMicrosoftprojectreunion].  The installer adds both the WinUI 3 project templates and the NuGet package containing the WinUI 3 libraries to Visual Studio 2019.  
-    
-    For instructions on how to add the `VSIX` package to Visual Studio, navigate to [Finding and Using Visual Studio Extensions][VisualstudioIdeFindingUsingVisualStudioExtensionsInstallWithoutUsing-ManageExtensionsDialogBox].
-    
+1. Follow steps 1-4 of the [WindowsAppSDK environment setup guide][WindowsAppsWinui3ConfigureYourDevEnvironment]. This guide will walk you through minimum system requirements, installing visual studio, enabling NuGet package source, and installing the WindowsAppSDK Extension for Visual Studio. 
+1. Install [WebView2 Runtime][Webview2Installer] or any [Microsoft Edge (Chromium) non-stable channel][MicrosoftedgeinsiderDownload] installed on Windows 10 version 1803 \(build 17134\) or later.  For more information about Windows 10, navigate to [Windows Update: FAQ][MicrosoftSupport12373].  
 1.  To access all developer-specific Visual Studio features, turn on [Developer Mode][WindowsUwpGetStartedEnableYourDeviceForDevelopment].  
     
 ## Step 1 - Create Project  
```

Comment:
Step 2 says "choose C#, Windows, and WinUI respectively" but the image (winui-getting-started-selections.png) seems to show C#, Windows, Desktop, and WinUI. Do the text or image need to be changed to match each other?

Reply:
text can be changed to specify desktop

Reply:
Actually the drop down selections only have spots for C#, Windows, WinUI, there is no place to choose Desktop, In the highlighted selection that is just an extra tag. We can leave it as is for now.